docs/src/overview/build-deploy.md

```diff
@@ -1,118 +1,80 @@
+The container configuration for both your application and its underlying services is exclusively based on your configuration files.
+A container has a 1:1 relationship with a Git commit.
+This means that builds are repeatable and that if the system detects that there aren't changes for a container,
+it reuses the existing container image skipping the build step and saving time.
 
-The live environment is composed of multiple containers&mdash;both for your applications and for the services it depends on.
-It also has a virtual network connecting them as well as a router to route incoming requests to the appropriate application.
 
-Based on your application type, the system selects one of the pre-built container images and runs the following:
+## The deploy
+The deploy process makes those containers live, replacing the previous versions, with virtually no interruption in service.
 
-1. First, any dependencies specified in the `.platform.app.yaml` file are installed.
-   Those include tools like Sass, webpack, and Drupal Console.
-2. Then, depending on the build flavor specified in the configuration file, a series of standard commands is run.
-   The default for PHP containers, for example, is running `composer install`.
 
-3. Finally, the build hook from the configuration file is run.
-   The build hook comprises one or more shell commands that you write to finish creating your production code base.
-   That could be compiling Sass files, running a bundler, rearranging files on disk,
-   compiling an application in a compiled language, or whatever else you want.
-   Note that, at this point, all you are able to access is the file system; there are no services or other databases available.
+### Deploy steps
 
-Once all of that is completed, we freeze the file system and produce a read-only container image.
-That image is the final build artifact:
-a reliable, repeatable snapshot of your application, built the way you want, with the environment you want.
+Hold requests
+: The system holds incoming requests so there is no service interruption.
 
-Because container configuration (for both your application and its underlying services) is exclusively based on your configuration files
-and your configuration files are managed through Git,
-we know that a given container has a 1:1 relationship with a Git commit.
-That means builds are always repeatable.
-It also means that if we detect that there are no changes that would affect a given container,
-we can skip the build step entirely and reuse the existing container image, saving a great deal of time.
+Unmount current containers
+: The containers gets disconnected from their file system mounts.
 
-In practice, the entire build process usually takes less than a minute.
+Mount file systems
+: The containers that were disconnected in the previous step, connect the file system to the new containers.
```

Comment:
```suggestion
: The containers that were disconnected in the previous step connect the file system to the new containers.
```

I don't think the containers do the connecting. Perhaps something like "The new containers are connected to their file systems", but I'm uncertain about the intent here.